doc/API.md

```diff
@@ -100,16 +100,16 @@ Returns a list of all rules with their ID and description
 
 The current set of tags supported are listed in the following table:
 
-| Tag Name        |       Accessibility Standard/Purpose        |
-| --------------- | :-----------------------------------------: |
-| `wcag2a`        |         WCAG 2.0 & WCAG 2.1 Level A         |
-| `wcag2aa`       |        WCAG 2.0 & WCAG 2.1 Level AA         |
-| `wcag21a`       |              WCAG 2.1 Level A               |
-| `wcag21aa`      |              WCAG 2.1 Level AA              |
-| `section508`    |                 Section 508                 |
-| `best-practice` |      Best practices endorsed by Deque       |
-| `experimental`  |           Cutting-edge techniques           |
-| `cat`           | Category mappings used by Deque (see below) |
+| Tag Name        | Accessibility Standard / Purpose            |
+| --------------- | ------------------------------------------- |
+| `wcag2a`        | WCAG 2.0 Level A                            |
+| `wcag2aa`       | WCAG 2.0 Level AA                           |
+| `wcag21a`       | WCAG 2.1 Level A                            |
+| `wcag21aa`      | WCAG 2.1 Level AA                           |
+| `best-practice` | Common accessibility best practices         |
+| `section508`    | Old Section 508 rules                       |
```

Comment:
Should we be calling this Old?

Reply:
Yeah, this is how we're referring to it in other products as well.